app/views/accounts/_email.html.slim

```diff
@@ -1,13 +0,0 @@
-.clearfix.border-top.border-blue-light
```

Comment:
This file was a copy of the webauthn template. It is not used anywhere so I removed it.